scripts/dockerimages/scripts/console.sh

```diff
@@ -82,6 +82,14 @@ if ! grep -q '^UseDNS no' /etc/ssh/sshd_config; then
     echo "UseDNS no" >> /etc/ssh/sshd_config
 fi
 
+if ! grep -q '^PermitRootLogin no' /etc/ssh/sshd_config; then
+    echo "PermitRootLogin no" >> /etc/ssh/sshd_config
+fi
+
+if ! grep -q '^AllowUsers rancher' /etc/ssh/sshd_config; then
```

Comment:
@janeczku see https://github.com/rancherio/os/commit/9925dccb087d717c21674718359db4728e94ae2e, changed to AllowGroups instead so it can be managed by user groups.